.github/workflows/aws.yml

```
@@ -28,7 +28,7 @@ name: Deploy to Amazon ECS
 
 on:
   push:
-    branches: ["main"]
+    branches: ["main","dev"]
```

YAMLlint (1.37.1)

[error] too few spaces after comma

(commas)